### PR TITLE
 appear the sign up form when no user

### PR DESCRIPTION
**Pull request by salsabeelomar**
appear the sign up form when no user

### CODE REVIEW

**Review by Saeed99Madi — APPROVED** [review]

**Review by dev-mohmmed-alshorafa — APPROVED** [review]

**Review by AseelL — APPROVED** [review]

File: client/src/components/Profile/Calender/Calender.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const Calender = () => {
     title: '',
     description: '',
   });
-  const [open, setOpen] = useState<IOpen>({
+  const [open, setOpenPop] = useState<IOpen>({
```
Comment (Amoodaa) [inline client/src/components/Profile/Calender/Calender.tsx:26]: rename the `open` to `openPop`

Comment (Amoodaa) [reply client/src/components/Profile/Calender/Calender.tsx:26]: or destructure it

**Review by Amoodaa — CHANGES_REQUESTED** [review]
Change var

File: client/src/components/Profile/Calender/Calender.tsx (hunk 1 above)
Comment (Saeed99Madi) [reply client/src/components/Profile/Calender/Calender.tsx:26]: i think this is my code :+1: 
can i take this ?

**Pull request merged**